build.cake

```diff
@@ -0,0 +1,32 @@
+var target = Argument("target", "Build");
+
+Task("Clean")
+	.Does(() => 
+{
+	var settings = new DotNetCoreCleanSettings
+     {
+         Framework = "netstandard2.0",
```

Comment:
Is this needed here? Isn't what's defined in the project enough?

Reply:
Wanted to use the overload with settings, sort of as an example, not needed, if you want I can remove it

Reply:
Not urgent, but if we change it in the project, then we have to remember to change it here as well, if it's not needed, better 🙂